API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
```

Comment:
Will this be a full locale or only language? I'm guessing the former, but we should work out how we're going to show available activities in a supported language, regardless of locale.

Example: let's say we have Activities in en-US and a new one written in en-GB. Do we show en-GB just that one, or all English ones? Not necessarily a concern for the model, but something to ponder for API design.